jmix-bom/bom.gradle

```diff
@@ -279,14 +280,77 @@ dependencies {
 
         api 'com.amazonaws:aws-java-sdk-core:1.11.1034'
 
-        def yargVersion = '2.2.21'
-        api "com.haulmont.yarg:yarg-api:$yargVersion"
-        api "com.haulmont.yarg:yarg:$yargVersion"
+        api 'org.apache.httpcomponents:httpclient:4.5.14'
+
+        api 'commons-logging:commons-logging:1.2'
+        api 'org.apache.commons:commons-lang3:3.9'
+        api 'commons-io:commons-io:2.6'
+        api 'org.dom4j:dom4j:2.1.3'
+        api 'com.google.guava:guava:28.0-jre'
+        api 'org.slf4j:slf4j-api:1.7.26'
+        api 'org.codehaus.groovy:groovy-all:2.4.12'
```

Comment:
Why org.codehaus.groovy with old version while we use new org.apache.groovy with the version from spring boot dependencies?